app/controllers/admin/facilities_controller.rb

```diff
@@ -89,7 +89,10 @@ def facility_params
       :longitude,
       :enable_diabetes_management,
       :monthly_estimated_opd_load,
-      :zone
+      :zone,
+      :enable_teleconsultation,
+      :teleconsultation_phone_number,
+      :teleconsultation_isd_code
```

Comment:
If the `teleconsultation_phone_number` is blank, should we also clear out the `teleconsultation_isd_code`? Otherwise every update to a facility will persist an ISD code behind the scenes.